### ISSUE TITLE
wording of new "trigger conditions"

### ISSUE THREAD

**Issue by andreineculau** [tomasbjerre/pull-request-notifier-for-bitbucket#68]
Thanks for fixing #64 ! The wording and the positioning (before `Triggers`) could be improved a bit though from:

```
Trigger if pull request
* Has, or has no, merge conflicts
* Has no merge conflicts
* Has merge conflicts

Triggers
...
```

to:

```
Triggers
...

Triggers apply
* always
* to PRs with merge conflicts
* to PRs without merge conflicts
```

and in accordance with #71 , it should actually be

```
Triggers
...

Triggers do not apply to
[ ] PRs with merge conflicts
[ ] PRs without merge conflicts
[ ] open PRs
[ ] merged PRs
[ ] declined PRs
```

**Comment by tomasbjerre** [comment]
Changed texts in this issue, continuing discussion in https://github.com/tomasbjerre/pull-request-notifier-for-bitbucket/issues/71

**Comment by tomasbjerre** [comment]
Released in 2.4. So only available for Bitbucket 4.

**Comment by tomasbjerre** [comment]
and in 1.30:
https://github.com/tomasbjerre/pull-request-notifier-for-bitbucket/releases/tag/pull-request-notifier-for-stash-1.30
I may not release that one to Marketplace:
https://answers.atlassian.com/questions/30935697/how-do-i-maintain-plugin-compatible-with-stash-3-and-bitbucket-4
